experimental/client-csharp/examples/Simple/Program.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Wandb.Sdk;
+using System.Collections.Generic;
+
+// See https://aka.ms/new-console-template for more information
+Console.WriteLine("Hello, World!");
+
+var wandb = new WandbClient();

```

Comment:
Could make it `IDisposable` instead and support the `using` syntax, similar to Python's `with`.